ibm_catalog.json

```diff
@@ -1,156 +1,241 @@
-                                    "type": "image/svg+xml"
-                                },
-                                "description": "A single zone OCP VPC cluster configured with KMS encryption for cluster data and worker block storage using a KMS encryption key from another account."
-                            }
-                        ]
-                    }
+                  "displayname": "RHCOS",
+                  "value": "RHCOS"
                 }
+              ]
+            },
+            {
+              "key": "ocp_entitlement"
+            },
+            {
+              "key": "number_worker_nodes"
+            },
+            {
+              "key": "machine_type"
+            },
+            {
+              "key": "vpc_id"
+            },
+            {
+              "key": "existing_cos_id",
+              "required": true
+            },
+            {
+              "key": "additional_lb_security_group_ids"
+            },
+            {
+              "key": "additional_vpe_security_group_ids"
+            },
+            {
+              "key": "addons"
+            },
+            {
+              "key": "allow_default_worker_pool_replacement"
+            },
+            {
+              "key": "attach_ibm_managed_security_group"
+            },
+            {
+              "key": "cbr_rules"
+            },
+            {
+              "key": "cluster_config_endpoint_type"
+            },
+            {
+              "key": "cluster_ready_when"
+            },
+            {
+              "key": "custom_security_group_ids"
+            },
+            {
+              "key": "disable_outbound_traffic_protection"
+            },
+            {
+              "key": "disable_public_endpoint"
+            },
+            {
+              "key": "enable_ocp_console"
+            },
+            {
+              "key": "force_delete_storage"
+            },
+            {
+              "key": "ignore_worker_pool_size_changes"
+            },
+            {
+              "key": "import_default_worker_pool_on_create"
+            },
+            {
+              "key": "kms_config"
+            },
+            {
+              "key": "manage_all_addons"
+            },
+            {
+              "key": "number_of_lbs"
+            },
+            {
+              "key": "pod_subnet_cidr"
+            },
+            {
+              "key": "use_private_endpoint"
+            },
+            {
+              "key": "service_subnet_cidr"
+            },
+            {
+              "key": "verify_worker_network_readiness"
+            },
+            {
+              "key": "worker_pools_taints"
+            },
+            {
+              "key": "resource_tags"
+            },
+            {
+              "key": "access_tags"
+            }
+          ],
```

Comment:
Please review the sorting of the variables and group related ones together

- VPC / network related
- Cluster related
- COS related
- KMS related

You get the picture